src/main/python/core/command_line_parser.py

```diff
@@ -29,17 +29,21 @@ def _create_argument_parser(self):
                               nargs=1,
                               type=str,
                               required=True,
-                              help='The location of the CADD'
-                                   ' annotated SNV file.')
+                              help='The location of the converted VEP'
+                                   ' annotated TSV file.')
 
         optional.add_argument('-o',
```

Comment:
`-o CAPICE_output.tsv.gz` should write the output file in the directory from which command-line call was made instead of `/capice/CAPICE_example/`

Reply:
Fixed in #54

Reply:
Resolving conversation.